_posts/2024-04-15-trino-fest-2024-approaches.md

```diff
@@ -0,0 +1,124 @@
+practices for running Trino on S3 data lakes - no matter which storage system
+you're using. It's a great opportunity to learn about how you can scale
+workloads, optimize data layouts, or reduce latency when using S3 for your data
+lake storage.
+
+## Data takes
+
+Of course, there's more to Trino than querying data lakes, and there's a wide
+variety of talks to discuss the other activities going on within the Trino
+community.
+
+### Small scale
+
+Ben Jeter at Executive Homes, who gave [a talk at Trino Fest last year]({{site.baseurl}}{% post_url 2023-07-25-trino-fest-2023-datto %})
+while at Datto, is back to discuss running Trino at a more moderate scale than
+that we're used to hearing about in the Trino space. Forget petabytes and
+exabytes, and welcome a tiny cluster querying thousands (not millions) of
+records that still derives huge value from Trino. It's a great playbook for
+smaller startups and enterprises who still need robust, performant analytics.
+
+### Maximizing performance
+
+Jonas Kylling from Dune Analytics will be detailing how they've managed to
+optimize Trino and squeeze out every ounce of performance to reduce query costs
+and runtimes. That includes leveraging the new Alluxio-based file system
+caching, emulating various cluster sizes to avoid expensive idle cluster time,
+and storing, sampling, and filtering query results to avoid re-executing
+queries.
+
+### Query intelligence
+
+Marton Bod at Apple brings insights to query intelligence. He'll demonstrate how
+Apple has understood when their clusters are most utilized and who's using them,
+enabling slicing and dicing along different dimensions. Having a query
+intelligence dataset can be used for real-time cluster dashboarding,
+self-service troubleshooting, and automatic generation of recommendations for
+users, all of which can empower Trino to be better than ever.
+
+## And more!
+
+Of course, Trino's own Martin Traverso will be giving a keynote on the latest
+and greatest in the project, covering everything big that's landed since Trino
+Summit, as well as a glimpse at the roadmap for the project in the coming few
+months. Several other big talks are falling into place that we can't announce
+just yet, so stay tuned for more info as the event draws nearer.
+
+## <a name="tcc"/> Trino contributor congregation
+
+The day after Trino Fest, we'll also be hosting an in-person meetup for
+Trino contributors and engineers to catch up, discuss the Trino roadmap, and
+engage directly with the maintainers in-person. It's a great opportunity to put
+faces and voices to those GitHub handles, align on the big ideas or tricky PRs
+that have been moving slowly, and find more ways to get involved in Trino
+development. If you're interested in attending, message Manfred Moser or Cole
+Bowden on the [Trino Slack]({{site.url}}/slack.html), and we'll get you added to
+the attendee list and share more details.
```

Comment:
Need to add a section about the sponsors with the logos at the bottom here .. or maybe higher up.. but somewhere